common/fixation_entry.go

```diff
@@ -233,7 +237,14 @@ func (fs *FixationStore) AppendEntry(
 		fs.setEntryIndex(ctx, safeIndex, true)
 	} else {
 		if block < latestEntry.Block {
-			panic(fmt.Sprintf("AppendEntry for block %d < latest entry block %d", block, latestEntry.Block))
+			// how come getUnmarshaledEntryForBlock lied to us?!
+			utils.LavaFormatError("critical: AppendEntry block smaller than latest",
```

Comment:
why are we not returning? is it safe to continue without returning?

Reply:
should we keep this panic?
or return an error

Reply:
good catch - should return.
fixed d71c4644

Reply:
you didn't return?
In the commit, it's still the same

Reply:
I cited the wrong commit - now it's the right commit reference.